ui/src/tool-panel/location-track/splitting/location-track-splitting-infobox.tsx

```diff
@@ -392,6 +408,15 @@ export const LocationTrackSplittingInfobox: React.FC<LocationTrackSplittingInfob
         (dupe) => dupe.draftType !== 'OFFICIAL',
     );
 
+    const postSplit = () => {
+        startPostingSplit();
+        postSplitLocationTrack(
+            splitRequest(locationTrack.id, initialSplit, splits, duplicateTracksInCurrentSplits),
+        )
+            .then(() => stopSplitting())
+            .catch(() => returnToSplitting());
+    };
```

Comment:
Tässä käytännössä koko putki sille mitä tehdään nyt kun operaattori painaa "Suorita jako" -nappulaa:
1. Asetetaan frontin sisäisesti splittauksen tila siihen, että sitä ollaan lähettämässä bäkkärille
2. Tehdään splittaus bäkkärille
3. Jos kaikki meni ok -> poistutaan splittaustilasta
4. Jos jotain meni pieleen -> palataan takaisin splittaustilaan ja mahdollistetaan muokkaaminen

Reply:
Kuvaisin itse mielummin tuon tilana (`setX(x)` tms) kuin siirtymänä (`returnToX`). Silloin tarvitsee vain yhden funktion eikä tilojen kombinaatioihin (jos sellaisia joskus tulee) tarvitse ottaa kantaan funktionimissä tai  miettiä onko joku tietty siirtymä mahdollinen tietyssä tilassa. Eli tässä tapauksessa pitäisin parempana `setPostInProgress(boolean)` kuin `startPostingSplit`/`returnToSplitting`. Lisäksi kuten aiemmin mainitsin, tuo tila sopisi minusta paremmin tänne komponenttiin kuin reduxiin, sillä tällainen in-flight tieto on luonteeltaan transienttia -> tallennettu arvo voi helposti olla väärä.